File: JavaWebProfession/JavaFundamentals/JavaAdvanced/JavaAdvancedExercises/JavaSyntax/src/TransportPrice.java
import java.util.Scanner;

public class TransportPrice {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        int distance = Integer.parseInt(console.nextLine());
        String timeOfDay = console.nextLine();

        double price = 0;

        if (distance < 20) {
            price += 0.7;

            if (timeOfDay.equals("day")) {
                price = price + distance * 0.79;
            } else {
                price = price + distance * 0.9;
            }
        } else if (distance < 100) {
            price = distance * 0.09;
        } else {
            price = distance * 0.06;
        }

        System.out.printf("%.2f", price);
    }
}

File: BasicModule/ProgrammingBasics/ProgrammingBasicsExams/Coding101March6/02.TransportPrice/TransportPrice.cs
using System;

class TransportPrice
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        string timeOfDay = Console.ReadLine();

        if (n < 20)
        {
            if (timeOfDay == "day")
            {
                Console.WriteLine(0.70 + 0.79 * n);
            }
            else
            {
                Console.WriteLine(0.70 + 0.90 * n);
            }
        }
        else if (n < 100)
        {
            Console.WriteLine(0.09 * n);
        }
        else
        {
            Console.WriteLine(0.06 * n);
        }
    }
}